### ISSUE TITLE
Error on install

### ISSUE THREAD

**Issue by operatorone** [1up-lab/contao-email-login#2]
Hi, if I install this package in a Contao 4.4.12 I get the following error message:

```
In ContaoModuleBundle.php line 36:                            
  The module folder "system/modules/!composer" does not exist. 
```

Tell me. If you need anyy more info.

Cheers,
operatorone

**Comment by bytehead** [comment]
Thanks for reporting, will fix in a new release for Contao 4.x

**Comment by operatorone** [comment]
Thanks for the response.

**Comment by bytehead** [comment]
Fixed in 9d66e5c11033e35546eac4490636d88bbb7f44c0. A new release `1.2.1` is available.

**Comment by Kahmoon** [comment]
Is this extension still working with Contao 4.12.x? (Last update 2018)

**Comment by bytehead** [comment]
It should theoretically still work (used hook is still available in 4.12.x). I don't know if it's actually installable under 4.12. Can you check it out? I can provide a fix if needed.

**Comment by Kahmoon** [comment]
I installed it and login is working. No errors so far :-D
